### PR TITLE
E2E-13 Course search for Anonymous user

### PR DESCRIPTION
**Pull request by UvgenGen**
[E2E-13](https://youtrack.raccoongang.com/issue/E2E-13) Course search for Anonymous user

### CODE REVIEW

File: regression/pages/lms/index.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import os
+
+from edxapp_acceptance.pages.lms.index import IndexPage
+from regression.pages.lms import LMS_BASE_URL, LMS_PROTOCOL
+
+LOGIN_BASE_URL = '{}://{}'.format(LMS_PROTOCOL, 'edx-base-ficus-theme.raccoongang.com')
```
Comment (cmltaWt0) [inline regression/pages/lms/index.py]: `edx-base-ficus-theme.raccoongang.com` should be moved to a configuration.

File: regression/tests/lms/test_course_search_for_anon.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    def setUp(self):
+        super(CourseSearchAnonymous, self).setUp()
+        self.index_page = IndexPageExtended(self.browser)
+        self.COURSE_DISPLAY_NAME = os.environ.get('COURSE_DISPLAY_NAME', 'demo course')
+
+    def test_course_search_anonymous(self):
+        self.index_page.visit()
+        self.index_page.wait_for_page()
+        self.index_page.search(self.COURSE_DISPLAY_NAME)
+        self.index_page.wait_for_element_visibility('.courses','')
```
Comment (cmltaWt0) [inline regression/tests/lms/test_course_search_for_anon.py]: Missing space in `('.courses','')`

File: regression/tests/lms/test_course_search_for_anon.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        self.index_page.wait_for_element_presence('.active-filter','')
```
Comment (cmltaWt0) [inline regression/tests/lms/test_course_search_for_anon.py]: The same as above.

**Review by cmltaWt0 — CHANGES_REQUESTED** [review]

File: regression/pages/lms/index.py (hunk 1 above)
Comment (UvgenGen) [reply regression/pages/lms/index.py]: changed

File: regression/tests/lms/test_course_search_for_anon.py (hunk 2 above)
Comment (UvgenGen) [reply regression/tests/lms/test_course_search_for_anon.py]: changed

**Review by cmltaWt0 — APPROVED** [review]

**Pull request merged**